Core/Src/main.c

```diff
@@ -44,7 +46,8 @@
 /* Private variables ---------------------------------------------------------*/
 
 /* USER CODE BEGIN PV */
-
+const uint8_t START = 0b00000001;//start bit
+const uint8_t CONFIG[2] = {0b10000000, 0x00};//single-ended channel 0 SPI followed by 12 0 bits
```

Comment:
recommend you combine these two lines into one array of 3 uint8_t elements.

Reply:
Addressed :)